src/object.d

```diff
@@ -842,14 +842,14 @@ class TypeInfo_Vector : TypeInfo
 
 class TypeInfo_Function : TypeInfo
 {
-    override string toString() const
+    override string toString() const @trusted
```

Comment:
Doesn't that change the API ? Meaning all overrides need to be `@safe` (or `@trusted`) ?

Reply:
It doesn't because `TypeInfo.toString` is already `@safe`.